scripts/linters/general_purpose_linter.py

```diff
@@ -539,26 +543,35 @@ def check_bad_pattern_in_file(filepath, file_content, pattern):
             Object containing details for the pattern to be checked.
 
     Returns:
-        bool. True if there is bad pattern else false.
+        failed: bool. True if there is bad pattern else false.
```

Comment:
Please follow the general pattern for the return's doc string

Reply:
Done.